lib/helpers.php

```diff
@@ -149,7 +153,7 @@ function micropub_post($endpoint, $params, $access_token, $file_path = NULL, $js
     $mimetype = finfo_file($finfo, $file_path);
     $multipart = new p3k\Multipart();
     $multipart->addArray($params);
-    $multipart->addFile($file_prop, $file_path, $mimetype);
+    $multipart->addFile($file_prop, $filename, $mimetype, $file_content);
```

Comment:
I just published version 0.2. Can you set the version requirement of `p3k\multipart` to `">=0.2.0"` in `composer.json`?